stackstorm-menandmice/etc/action_template.yaml.j2

```diff
@@ -0,0 +1,43 @@
+---
+description: "{{ operation_description }}"
```

Comment:
Just want to confirm that the value passed as description and other values are correctly escaped (e.g. in case description or other field contains double quotes, they are escaped).